api-docs/events.md

```diff
@@ -0,0 +1,45 @@
+# Events
+
+The list of events will document CU events.
+
+## Get single Event
+
+### Request
+
+```http
+GET /api/event/:id
```

Comment:
Where will this API be used in the UI?

Reply:
If I followed the events part of the conversation right eventually it will be an aspect of the scheduling phase, but for this phase they only need to be able to add basic event data so it can be associated with the leads generated from the event. Maybe a modal form on the leads page for now?